src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -474,7 +480,10 @@ export const validateDisabilityName = (
     item => item === itemLowerCased || sippableId(item) === itemSippableId,
   );
   if (itemCount.length > 1) {
-    err.addError('Please enter a unique condition name');
+    const errorMessage = showRevisedNewDisabilitiesPage()
+      ? "You've already added this condition to your claim"
```

Comment:
```suggestion
      ? "You’ve already added this condition to your claim"
```

see note [here](https://design.va.gov/content-style-guide/punctuation#apostrophes) about curly apostrophes